config/scheduler.yml

```diff
@@ -2,25 +2,42 @@ monitor_queue_sizes:
   cron: "* * * * * America/New_York"
   class: "MonitorJob"
   queue: monitor
-  args: [ queue_sizes, ignore_and_log_errors ]
+  args:
+    - "Monitoring::QueueSizeCheck"
   description: "Send queue sizes to Librato"
+monitor_queue_sizes_aws:
```

Comment:
@jlambert121 here's what seemed simplest to me.  I know it would hit redises twice for the same info, but it's not a huge burden and keeps things simple (ish).

This says "check using the queue size checker and notifier using the AwsNotifier with the given configuration"